src/languages/en-US/commands/management.json

```diff
@@ -356,10 +356,10 @@
 			"threshold-duration 30s"
 		]
 	},
-	"filterDescription": "Manage this guild's word blacklist.",
+	"filterDescription": "Manage this guild's blocked word list.",
 	"filterExtended": {
 		"usages": ["add Word", "remove Word", "reset", "show", ""],
-		"extendedHelp": "The filter command manages the word blacklist for this server and must have a filter mode set up, check `Skyra, help filterMode`.\n\nSkyra's word filter can find matches even with special characters or spaces between the letters of a blacklisted word, as well as it filters duplicated characters for enhanced filtering.",
+		"extendedHelp": "The filter command manages the blocked word list for this server and must have a filter mode set up, check `Skyra, help filterMode`.\n\nSkyra's word filter can find matches even with special characters or spaces between the letters of a blocked word, as well as it filters duplicated characters for enhanced filtering.",
```

Comment:
```suggestion
		"extendedHelp": "The filter command manages the list of blocked words for this guild and must have a filter mode set up (see: `Skyra, help filterMode`).\n\nSkyra's word filter can find matches even with special characters or spaces between the letters of a blocked word, as well as duplicated characters for enhanced filtering.",
```

Reply:
We use "server" instead of "guild" as it's the term used in Discord's UI, can you revert that word change, please?

Reply:
On a second thought, I'll rename all references in a later PR, I'll accept this as-is.